Gemfile

```diff
@@ -1,12 +1,14 @@
 source 'https://rubygems.org'
 
-gem 'berkshelf'
-gem 'thor-foodcritic'
+gem 'berkshelf', '~> 2.0.18'
```

Comment:
I will be in the future, I am just trying to get travis to complete now. So I can merge the PR's